katana_requirements.yaml

```diff
@@ -111,8 +111,10 @@ cmake:
     - apt
     - conda/dev
   version_overrides:
-    conda/dev: ">=3.17,<3.22a0"
-    conda: ">=3.17,<3.22a0"
+    # This unfortunate mess is due to the fact that apt doesn't have a CMake 3.22.3 package available yet
+    # TODO (scober)(KAT-3901): bump the minimum CMake version to 3.22.3
+    conda: ">=3.17,<4a0,!=3.22.1,!=3.22.2"
+    conda/dev: ">=3.17,<4a0,!=3.22.1,!=3.22.2"
```

Comment:
You could override the apt version instead of messing up conda. We are dropping support for APT anyway.

Reply:
Good idea